### ISSUE TITLE
Improper user name in HipChat

### ISSUE THREAD

**Issue by lacombar** [cdidier/irssi-xmpp#9]
When starting a 1-on-1 conversation in HipChat, the following show up:

```
01:22 <abcdef_ghijklm@chat.hipchat.com/web||proxy|proxy-c300.hipchat.com|5262> test
```

This makes `irssi-xmpp` unusable. For reference, Pidgin is capable to providing the correct "clear" username, eg. "Foo Bar"

**Comment by ailin-nemui** [comment]
irssi-xmpp is currently lacking an option to display anything else than the raw JID for queries

**Comment by lacombar** [comment]
is there any reason for this limitation ? this seems to works fine in rooms.

**Comment by ailin-nemui** [comment]
not really, in XMPP rooms irssi-xmpp will show the user room nick. HipChat rooms set the user nicks to what you expect to see.
